testsuite/src/it/java/org/graalvm/tests/integration/AppReproducersTest.java

```diff
@@ -551,6 +551,12 @@ public void imageioAWT(TestInfo testInfo, Apps app) throws IOException, Interrup
             controlData.keySet().forEach(f -> new File(appDir, f).delete());
 
             LOGGER.info("Running...");
+
+            // Fontconfig might look for fonts here, see similar thing in Quarkus:
+            // https://github.com/quarkusio/quarkus/blob/main/extensions/awt/runtime/src/main/java/io/quarkus/awt/runtime/JDKSubstitutions.java#L53
+            Files.createDirectories(Path.of(appDir.toString(), "conf", "fonts")).toFile().deleteOnExit();
+            Files.createDirectories(Path.of(appDir.toString(), "lib")).toFile().deleteOnExit();
```

Comment:
I don't think we need both. One of the two should be sufficient. See:
https://github.com/Karm/mandrel-integration-tests/issues/151
and
https://github.com/openjdk/jdk17u/blob/908cab4123812b6b206b966a6ce92398cdf42c08/src/java.desktop/share/classes/sun/awt/FontConfiguration.java#L183 and https://github.com/openjdk/jdk17u/blob/908cab4123812b6b206b966a6ce92398cdf42c08/src/java.desktop/share/classes/sun/awt/FontConfiguration.java#L193

I suggest to only create the `lib` directory.